src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksBatchUtilityServiceImpl.java

```diff
@@ -70,7 +72,21 @@ public String getFileContents(String fileName) {
     }
 
     @Override 
-    public Note createNote(String noteText) {
+    public KfsVendorDataWrapper createNoteRecordingAnyErrors(KfsVendorDataWrapper kfsVendorDataWrapper, String noteText, String noteErrorDescriptor) {
+        if (noteTextSizeIsWithinLimit(noteText)) {
+            Note note = createNote(noteText);
+            kfsVendorDataWrapper.getVendorNotes().add(note);
+        } else {
+            LOG.error("createNoteRecordingAnyErrors: The " + noteErrorDescriptor + " Note contained " + noteText.length() + "characters and it can only have " + PaymentWorksConstants.NOTE_TEXT_DEFAULT_MAX_LENGTH);
```

Comment:
do you think it would be helpful to add the note text to the log?

Reply:
Some of the note context is restricted by user security so even though it would help in debugging, I do not believe we should include it.

Reply:
OK